instrumentation/grape/lib/opentelemetry/instrumentation/grape/instrumentation.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Grape
+      # The Instrumentation class contains logic to detect and install the Grape instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
+        MINIMUM_VERSION = Gem::Version.new('1.2.0')
```

Comment:
I set the minimum to 1.2.0 because of this: https://github.com/DataDog/dd-trace-rb/blob/a02ccfc38123c087451f204ac6e822a6240216d5/lib/datadog/tracing/contrib/grape/endpoint.rb#L197
Since 1.2.0 is already quite old, I didn't think it was worth it to add that condition to ensure compatibility with older versions.

Reply:
I think the bug that was mentioned is fixed in `1.2.2` but that has to be checked.

Reply:
1.2.0 should be good because that's when [this change](https://github.com/ruby-grape/grape/blob/3cf89cf1f253a36c73d4b9e8a6b2aca673f0b915/UPGRADING.md#changes-in-the-grapeapi-class) was introduced. From 1.2.0 onwards we can use `Grape::API::Instance`.
Indeed this caused a bug in another part of the framework that was solved in 1.2.2, but as far as our instrumentation is concerned, 1.2.0 should be enough to avoid the workaround 😄